app/components/windows/Projector.vue.ts

```diff
@@ -8,7 +8,7 @@ import { ISourcesServiceApi } from 'services/sources';
 import electron from 'electron';
 import Util from 'services/utils';
 import { $t } from 'services/i18n';
-import { Subscription } from 'rxjs/subscription';
+import { Subscription } from 'rxjs';
```

Comment:
I've seen this imported as `rxjs/Subscription` in other places. Could we keep it case consistent?